keyboards/ah/haven65/keymaps/via/config.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2023 Freather
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+
+#define EECONFIG_USER_DATA_SIZE 4
```

Comment:
The default eeconfig size is 4 (and a bit simpler).  So if you don't need more than this ...

Reply:
this file has been removed